docs/release-notes/release-notes-0.17.0.md

```diff
@@ -12,9 +12,15 @@
   wtdb.BackupIDs](https://github.com/lightningnetwork/lnd/pull/7623) instead of 
   the entire retribution struct. This reduces the amount of data that needs to 
   be held in memory. 
+ 
 * [Replace in-mem task pipeline with a disk-overflow
   queue](https://github.com/lightningnetwork/lnd/pull/7380)
  
+* [Add defaults](https://github.com/lightningnetwork/lnd/pull/7771) to the 
+  wtclient and watchtower config structs and use these to populate the defaults 
+  of the main LND config struct so that the defaults appear in the `lnd --help` 
```

Comment:
Should mention removal of deprecated field here.

Reply:
good catch. adding that now!